openshift2nulecule/openshift.py

```diff
@@ -117,7 +117,8 @@ def export_project(self):
                              "persistentvolumeclaims",
                              "services"]
             elif provider == "openshift":
-                resources = ["all"]
+                resources = ["service", "deploymentConfig", "buildConfig",
```

Comment:
@kadel totally forgot about this, exporting `rc` was caused some problems BTW. The problem was the `mlbparks` app had two `rc` and one `rc` had information of other, so while importing information of one `rc` was sufficient but when tried to read the other `rc` info it would fail saying the `rc` already exists.